source/server/http/admin.cc

```diff
@@ -698,8 +698,7 @@ AdminImpl::AdminImpl(const std::string& access_log_path, const std::string& prof
           {"/runtime", "print runtime values", MAKE_ADMIN_HANDLER(handlerRuntime), false, false},
           {"/hystrix_event_stream", "send hystrix event stream",
            MAKE_ADMIN_HANDLER(handlerHystrixEventStream), false, false}},
-      listener_stats_(
-          Http::ConnectionManagerImpl::generateListenerStats("http.admin.", listener_scope)) {
```

Comment:
We changed that?

Reply:
no, it is their change that in the merge was lost. one of the reasons the previous PR did not compile.